docs/source/config_file.rst

```diff
@@ -666,6 +666,16 @@ section of the command line docs.
     Shows a warning when encountering any code inferred to be unreachable or
     redundant after performing type analysis.
 
+.. confval:: deprecated_calls_exclude
+
+    :type: comma-separated list of strings
+
+    Selectively excludes functions and methods defined in specific packages,
+    modules, and classes from the :ref:`deprecated<code-deprecated>` error code.
+    This also applies to all submodules of packages (i.e. everything inside
+    a given prefix). Note, this option does not support per-file configuration,
```

Comment:
Maybe end with ";" instead of ","?

Reply:
Tbh I mostly copied the contents from the `untyped_calls_exclude` option. So I'd leave it as is. https://mypy.readthedocs.io/en/stable/config_file.html#confval-untyped_calls_exclude